ci/test-conformance-eks.sh

```diff
@@ -253,6 +248,7 @@ function cleanup_cluster() {
 
 # ensures that the script can be run from anywhere
 THIS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+GIT_CHECKOUT_DIR=${THIS_DIR}/..
 pushd "$THIS_DIR" > /dev/null
```

Comment:
I don't know if this `pushd` call is still necessary now that we use `GIT_CHECKOUT_DIR` everywhere, but there is probably no harm in keeping it. Actually, given that we add this `pushd` call to start with, I am surprised that `GIT_CHECKOUT_DIR` was needed at all.

Reply:
I think it was after this `pushd` change the Jenkins builds start to encounter some problem with paths not found. In the Jenkins build step when it runs scripts rather than complete commands, it seems to copy the script to another directory and runs it from there.

Reply:
if that's the case then `GIT_CHECKOUT_DIR=${THIS_DIR}/..` may not work as well...

Reply:
I have tested that this works. So I'm assuming it will be run from a directory created under THIS_DIR.

Reply:
Ok, if it works it's what's important. Let me merge the PR then.